src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1223,11 +1309,17 @@ private Conversion ClassifyExplicitOnlyConversionFromExpression(BoundExpression
                     }
                 }
             }
+            else if (sourceExpression.GetFunctionType() is { })
+            {
+                if (IsValidFunctionTypeConversionTarget(destination, ref useSiteInfo))
+                {
+                    return Conversion.FunctionType;
```

Comment:
>return Conversion.FunctionType; [](http://example.com/codeflow?start=20&length=31)

Is this an implicit conversion? It feels like the name of the function implies that implicit conversions should be ignored.